control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -2471,7 +2471,16 @@ components:
     CreatePoolBody:
       example:
         disks:
-          - 'malloc:///disk?size_mb=100'
+          - "malloc:///disk?size_mb=100"
+        encryption:
+          data:
+            cipher: AES_XTS
+            key:
+              key_name: encrkey
+              key: [ 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 49, 50, 51, 52, 53, 54, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 100, 101, 97, 100, 99, 48 ]
```

Comment:
Should this be a string?

Reply:
Users are not supposed to use this directly. While creating a secret this will be a string. This is just for testing.